### PR TITLE
[DataFactory] Add 2 New commands to ADF V2

### PR DESCRIPTION
**Pull request by arsunda**
## Description

Adding 2 new commands: `Stop-AzDataFactoryV2TriggerRun` and `Invoke-AzDataFactoryV2TriggerRun`

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by isra-fel** [comment]
Hi @arsunda as you've probably seen in PR template, before introducing new cmdlets you need to submit a cmdelt design review (and make sure it's accepted) here: https://github.com/Azure/azure-powershell-cmdlet-review-pr

**Comment by arsunda** [comment]
> Hi @arsunda as you've probably seen in PR template, before introducing new cmdlets you need to submit a cmdelt design review (and make sure it's accepted) here: https://github.com/Azure/azure-powershell-cmdlet-review-pr

Hi @isra-fel , Ive added the Design Review here: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/749
Will someone be able to look at it over the next few days?

**Comment by arsunda** [comment]
@isra-fel The review has been accepted, can we check in this PR?

**Comment by arsunda** [comment]
@isra-fel ping on this

File: src/DataFactory/DataFactoryV2/help/Invoke-AzDataFactoryV2TriggerRun.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.DataFactoryV2.dll-Help.xml
+Module Name: Az.DataFactory
+online version:
```
Comment (isra-fel) [inline src/DataFactory/DataFactoryV2/help/Invoke-AzDataFactoryV2TriggerRun.md:4]:
```suggestion
online version: https://docs.microsoft.com/en-us/powershell/module/az.datafactory/invoke-azdatafactoryv2triggerrun
```

File: src/DataFactory/DataFactoryV2/help/Stop-AzDataFactoryV2TriggerRun.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.DataFactoryV2.dll-Help.xml
+Module Name: Az.DataFactory
+online version:
```
Comment (isra-fel) [inline src/DataFactory/DataFactoryV2/help/Stop-AzDataFactoryV2TriggerRun.md:4]:
```suggestion
online version: https://docs.microsoft.com/en-us/powershell/module/az.datafactory/stop-azdatafactoryv2triggerrun
```

**Review by isra-fel — CHANGES_REQUESTED** [review]
Please fix the online help issues detected by static analysis

**Review by isra-fel — APPROVED** [review]
LGTM

**Comment by isra-fel** [comment]
Pending #13237

**Comment by VeryEarly** [comment]
> Pending #13237
> Needs to re-run CI after it's merged

this build didn't get the latest common module 1.3.28 for some reason, it's using 1.3.26 instead.
re-triggered CI.

**Review by VeryEarly — DISMISSED** [review]

**Review by VeryEarly — APPROVED** [review]

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**